src/Exceptions/CallbackNotImplementException.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace PhpParallelLint\PhpParallelLint\Exceptions;
+
+class CallbackNotImplementException extends ParallelLintException
```

Comment:
```suggestion
class CallbackNotImplementedException extends ParallelLintException
```

Reply:
Thanks, fixed it in the rebase onto the updated master.